render_js.go

```diff
@@ -0,0 +1,279 @@
+// +build netgo
+
+package engo
+
+import (
+	"image/color"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+type Drawable interface {
+	Texture() *js.Object
+	Width() float32
+	Height() float32
+	View() (float32, float32, float32, float32)
+}
+
+type RenderComponent struct {
+	// Hidden is used to prevent drawing by OpenGL
+	Hidden bool
+
+	// Transparency is the level of transparency that is used to draw the texture
+	Transparency float32
+
+	scale  Point
+	Color  color.Color
+	shader Shader
+	zIndex float32
+
+	drawable      Drawable
+	buffer        *js.Object
+	bufferContent []float32
+}
+
```

Comment:
Looks like there's still a lot commented. Does this work even though it's commented?